<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="URL_OK">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="URL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>http://192.168.43.50:8080/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exercise_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exercise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rawData">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Raw Data Plot</string>
    </property>
   </widget>
   <widget class="RotatingCube" name="rotatingCube" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>251</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RotatingCube</class>
   <extends>QWidget</extends>
   <header>RotatingCube.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
